### PR TITLE
Next greater element in c

### PR DESCRIPTION
**Pull request by viveksondhiya**
## 🛠️ Issue (Number)

Issue no #376 

## ✔️ Check List (Check all the applicable boxes) 

- [ ✔️] My code follows the code style of this project.
- [✔️ ] This PR does not contain plagiarized content.

## 📷 Screenshots
image

### CODE REVIEW

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Give appropriate PR title

**Review by viveksondhiya — COMMENTED** [review]
corrections done

**Review by zugzwang03 — CHANGES_REQUESTED** [review]
add comments explaining approach

**Review by viveksondhiya — COMMENTED** [review]
Added the approach

**Review by zugzwang03 — APPROVED** [review]

**Review by Rishikavishnoi — APPROVED** [review]

**Review by 072arushi — APPROVED** [review]

File: C/Stack/nextgreaterelement.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
+// Approach for solving 
+# Initialize the stack with a dummy element with a value of -1.
```
Comment (Arun9739) [inline C/Stack/nextgreaterelement.c:2]: In C language, # will not be used for comments.
Plz put appropriate code.

**Review by Arun9739 — CHANGES_REQUESTED** [review]
@viveksondhiya Add screenshot of the output

**Review by viveksondhiya — COMMENTED** [review]
Changes done

**Review by Arun9739 — APPROVED** [review]

**Pull request closed without merging**